### ISSUE TITLE
NTR: retinal microaneurysm

### ISSUE THREAD

**Issue by cmungall** [obophenotype/human-phenotype-ontology#4265]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label

retinal microaneurysm

### 2. Synonyms

microneurysm of retina
retinal MA

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))

Abnormality of retinal vasculature; should group the two bolded terms in this image:

![image](https://user-images.githubusercontent.com/50745/48983916-be70c180-f0a9-11e8-884e-f3577c485dc2.png)

For potential additional subclasses see:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3943418/

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)

 * MONDO:0002311 ! retinal vascular disease
 * MONDO:0005015 ! diabetes mellitus (disease)
 * a wide range of systemic and retinal diseases, including ischemic, infectious, inflammatory, hematologic, and radiation-related disorders. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3943418/

Note that MONDO:0001257 retinal microaneurysm will be obsoleted in favor of an HPO class

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))

https://orcid.org/0000-0002-6601-2165

**Comment by pnrobinson** [comment]
@psergouniotis
-- it sounds like these two terms are (near) synonyms. Can you please comment on whether we need both or whether we should merge?

**Comment by pnrobinson** [comment]
@cmungall @psergouniotis
I have rearranged somewhat because retinal microaneurysms are mainly affecting capillaries and not retinal arteries (this would be a macroaneurysm).
